src/content/development/contribution-guide/_index.en.md

```diff
@@ -234,6 +235,19 @@ During the week when the general availability release is performed, the next ver
 Additionally, a retrospective meeting for the last release cycle will be held.
 There will be no dedicated [test day](#test-days), as the release candidate has been tested and no changes have occurred since.
 
+##### Release Notes
+
+[Release notes] should be published for each GA version.
+All new features and bug fixes of the release should be labeled with `release-note-needed` and have a relevant release note
+documented as part of the PR. Release notes submitted with PRs should be written in individual files in the `/release-notes` directory
+of the relevant repository. The release note file name should be prefixed with a date in the format `YYYYMMDD` and end with a `.md` suffix.
+The content of the file should be a Markdown snippets.
```

Comment:
Should we mention the files should have `<!-- markdownlint-disable MD041 -->` comment?

Otherwise the `markdownlint` fails because of missing main header

Reply:
Sounds good to add this